book.rb

```diff
@@ -7,3 +7,22 @@ def initialize(title, author)
     @rentals = []
   end
 end
+
+def create_book(books)
+  print 'Title: '
+  title = gets.chomp
+  print 'Author: '
+  author = gets.chomp
+  books << Book.new(title, author)
+  puts 'Book created successfully'
+end
```

Comment:
- [ ] The Single-Responsibility Principle of the `SOLID` object Oriented Design states that a class should have one job only. hence all requests for inputs should have been placed in the `main.rb` file /or `classes/methods` whose responsibilities are to request input. Please kindly work on this and move a step forward toward approval. 👌

  For more on the SOLID lessons, click on this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) 💡

> For example you can create a file called `require_inputs.rb` containing some functions responsible for requesting input from users 💡

```
// require_inputs.rb

def grap_book_data
    puts 'Title: '
    title = gets.chomp
    puts 'Author: '
    author = gets.chomp

  [title, author]
end
```
```
// book.rb ✅

def create_book(books)
    title, author = grap_book_data
    books << Book.new(title, author)
    puts 'Book created successfully'
end
```

Instead of 

```
// book.rb ❌

  def create_book(books)
    print 'Title: '
    title = gets.chomp
    print 'Author: '
    author = gets.chomp
    books << Book.new(title, author)
    puts 'Book created successfully'
  end
```

Reply:
Done! Thanks 😉